specs/IsSmartScreenEnabled

```diff
@@ -0,0 +1,87 @@
+# Background
+
+[Edge SmartScreen](https://docs.microsoft.com/en-us/deployedge/microsoft-edge-security-smartscreen) helps you identify reported phishing and malware websites,also helps you make informed decisions about downloads.
+
+In the current version of the Edge browser, users can switch the state of SmartScreen at any time. However, the webview2 application can only be set at initialization. In order to add more flexibility to the webview2 application. This API is added to keep the SmartScreen setting consistent with the browser. Development Users can control the status of SmartScreen at any time.
+
+In this document we describe the new setting. We'd appreciate your feedback.
+
+
+# Description
+The configuration of SmartScreen is stored in the user data folder. So its configuration is bound to the environment. Users can use ICoreWebView2Environment to control it.
+
+The default value for `IsSmartScreenEnabled` is `true`.
+When this setting is set to `false`, It will turn off SmartScreen protection when visiting web pages.
+
+Disabling/Enabling `IsSmartScreenEnabled` will take effect immediately.
+
+
+# Examples
+```cpp
+wil::com_ptr<ICoreWebView2Environment> enviroment;
+void SettingsComponent::ToggleSmartScreenEnabled()
```

Comment:
Does this sample code represent why an app would want to change IsSmartScreenEnabled? Why would an app want to change the value dynamically rather than just during initialization?

Reply:
From developer's perspective, they would like to have the possibility to turn on/off Smart Screen daynamically. In some scenarios, apps will provide toggle options for end-user to disable uploading any user privacy data. Thus developer may turn on/off accordingly based on end-users choice, which would be more flexible for developers.